src/lib/openapi/spec/project-status-schema.ts

```diff
@@ -14,6 +14,12 @@ export const projectStatusSchema = {
             description:
                 'Array of activity records with date and count, representing the project’s daily activity statistics.',
         },
+        averageHealth: {
+            type: 'number',
```

Comment:
Actually, now that I think about it, this can / should probably be an `integer` because we round it, right?

```suggestion
            type: 'integer',
```

And if not for this one, it at least goes for the `resources` properties - but that's on me. I'll fix it 🙋🏼